documentation/directives.md

```diff
@@ -3,6 +3,14 @@ title: "SDL directives"
 date: 2018-09-09T12:52:46+10:00
 description: How SDL Directives can be used to adjust the behavior of your graphql API
 ---
+# Directives
+
+[Directives](https://spec.graphql.org/draft/#sec-Language.Directives) are a powerful feature of GraphQL, allowing you to declare additional data to a schema or document. This data can then be used to change runtime execution or type validation behavior.
+
+There are two broad categories of directives, schema and operation directives. Schema directives are used on schema elements, and operation directives are used in operations within a GraphQL document.
```

Comment:
Unfortunately the whole Directives world is a bit confusing: to be very precise a Directive(definition) can be both if the Directive is allowed on Schema elements + Operation elements.

Therefore a Directive(definition) can be either or both.
An AppliedDirective (Directive instance) is always either one.

Not sure how or if we want to explain this here.

Reply:
I'll add a note so it's crystal clear

> Note for those who love the details: the terminology of directives is a bit confusing. It is technically possible to define a directive that is both a schema and operation directive, in other words, defined for both schema and operation locations. However in practice, this is not common.